docs/API.md

```diff
@@ -466,16 +477,13 @@ An object with the following properties:
 > TODO: Clarify the default value of each optional `"params"` property.
 >
 
-*priotity*
+priority parameter
```

Comment:
Please update the `"priority"` item in the list of params above, instead of having a separate section here.